### PR TITLE
Fix deb dependencies for qsql

### PR DESCRIPTION
**Pull request by poshul**
Add dependencies for qt5sql.

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: cmake/package_deb.cmake [hunk 1]
```diff
@@ -66,7 +66,7 @@ set(CPACK_COMPONENTS_ALL applications doc library share ${THIRDPARTY_COMPONENT_G
 
 ## Autogeneration with SHLIBDEPS will add to this variable. For now we include most things statically and require the standard Qt package only.
 ## (only available in Ubuntu >=17.10). For older Ubuntu, dependencies can be installed from a thirdparty repo.
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "qtbase5-dev (>= 5.7.0) | qt57base | qt58base | qt59base | qt510base | qt511base, libqt5svg5 (>= 5.7.0) | qt57svg | qt58svg | qt59svg | qt510svg | qt511svg")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "qtbase5-dev (>= 5.7.0) | qt57base | qt58base | qt59base | qt510base | qt511base, libqt5svg5 (>= 5.7.0) | qt57svg | qt58svg | qt59svg | qt510svg | qt511svg | libqt5sql5-sqlite | libsqlite3-dev")
```
Comment (jpfeuffer) [inline cmake/package_deb.cmake:69]: do we still need the package names for older ubuntus from third-party repos?

Comment (poshul) [reply cmake/package_deb.cmake]: 14.04 and 16.04 are still technically supported by Canonical, if it's not adding any overhead for us I'd suggest keeping them until those are EOL.

Comment (jpfeuffer) [reply cmake/package_deb.cmake]: ok

Comment (jpfeuffer) [reply cmake/package_deb.cmake]: I wonder how the plugin is called on the beineri repo tho

**Comment by poshul** [comment]
Are we good to move ahead on this?

**Comment by jpfeuffer** [comment]
Yes fine with me. Just note that is most likely will not work on older ubuntus anyway (see my comment about qsqliteplugin on the third-party PPA repo)

**Pull request merged**